packages/wallet/test/hdkey.spec.ts

```diff
@@ -9,108 +9,98 @@ const fixtureseed = hexToBytes(
-    )
-  })
+tape('.privateExtendedKey()', (t) => {
+  t.deepEqual(
+    fixturehd.privateExtendedKey(),
+    'xprv9s21ZrQH143K4KqQx9Zrf1eN8EaPQVFxM2Ast8mdHn7GKiDWzNEyNdduJhWXToy8MpkGcKjxeFWd8oBSvsz4PCYamxR7TX49pSpp3bmHVAY'
+  )
+  t.end()
 })
 
-describe('.publicExtendedKey()', function () {
-  it('should work', function () {
-    assert.strictEqual(
-      fixturehd.publicExtendedKey(),
-      'xpub661MyMwAqRbcGout4B6s29b6gGQsowyoiF6UgXBEr7eFCWYfXuZDvRxP9zEh1Kwq3TLqDQMbkbaRpSnoC28oWvjLeshoQz1StZ9YHM1EpcJ'
-    )
-  })
+tape('.publicExtendedKey()', (t) => {
+  t.deepEqual(
+    fixturehd.publicExtendedKey(),
+    'xpub661MyMwAqRbcGout4B6s29b6gGQsowyoiF6UgXBEr7eFCWYfXuZDvRxP9zEh1Kwq3TLqDQMbkbaRpSnoC28oWvjLeshoQz1StZ9YHM1EpcJ'
+  )
+  t.end()
 })
 
-describe('.fromExtendedKey()', function () {
-  it('should work with public', function () {
-    const hdnode = EthereumHDKey.fromExtendedKey(
-      'xpub661MyMwAqRbcGout4B6s29b6gGQsowyoiF6UgXBEr7eFCWYfXuZDvRxP9zEh1Kwq3TLqDQMbkbaRpSnoC28oWvjLeshoQz1StZ9YHM1EpcJ'
-    )
-    assert.strictEqual(
-      hdnode.publicExtendedKey(),
-      'xpub661MyMwAqRbcGout4B6s29b6gGQsowyoiF6UgXBEr7eFCWYfXuZDvRxP9zEh1Kwq3TLqDQMbkbaRpSnoC28oWvjLeshoQz1StZ9YHM1EpcJ'
-    )
-    assert.throws(function () {
-      hdnode.privateExtendedKey()
-    }, /^Error: No private key$/)
-  })
-  it('should work with private', function () {
-    const hdnode = EthereumHDKey.fromExtendedKey(
-      'xprv9s21ZrQH143K4KqQx9Zrf1eN8EaPQVFxM2Ast8mdHn7GKiDWzNEyNdduJhWXToy8MpkGcKjxeFWd8oBSvsz4PCYamxR7TX49pSpp3bmHVAY'
-    )
-    assert.strictEqual(
-      hdnode.publicExtendedKey(),
-      'xpub661MyMwAqRbcGout4B6s29b6gGQsowyoiF6UgXBEr7eFCWYfXuZDvRxP9zEh1Kwq3TLqDQMbkbaRpSnoC28oWvjLeshoQz1StZ9YHM1EpcJ'
-    )
-    assert.strictEqual(
-      hdnode.privateExtendedKey(),
-      'xprv9s21ZrQH143K4KqQx9Zrf1eN8EaPQVFxM2Ast8mdHn7GKiDWzNEyNdduJhWXToy8MpkGcKjxeFWd8oBSvsz4PCYamxR7TX49pSpp3bmHVAY'
-    )
-  })
+tape('.fromExtendedKey()', (t) => {
+  const hdnode1 = EthereumHDKey.fromExtendedKey(
+    'xpub661MyMwAqRbcGout4B6s29b6gGQsowyoiF6UgXBEr7eFCWYfXuZDvRxP9zEh1Kwq3TLqDQMbkbaRpSnoC28oWvjLeshoQz1StZ9YHM1EpcJ'
+  )
+  t.deepEqual(
+    hdnode1.publicExtendedKey(),
+    'xpub661MyMwAqRbcGout4B6s29b6gGQsowyoiF6UgXBEr7eFCWYfXuZDvRxP9zEh1Kwq3TLqDQMbkbaRpSnoC28oWvjLeshoQz1StZ9YHM1EpcJ'
+  )
+  t.throws(function () {
+    hdnode1.privateExtendedKey()
+  }, /^Error: No private key$/)
+  const hdnode2 = EthereumHDKey.fromExtendedKey(
+    'xprv9s21ZrQH143K4KqQx9Zrf1eN8EaPQVFxM2Ast8mdHn7GKiDWzNEyNdduJhWXToy8MpkGcKjxeFWd8oBSvsz4PCYamxR7TX49pSpp3bmHVAY'
+  )
+  t.deepEqual(
+    hdnode2.publicExtendedKey(),
+    'xpub661MyMwAqRbcGout4B6s29b6gGQsowyoiF6UgXBEr7eFCWYfXuZDvRxP9zEh1Kwq3TLqDQMbkbaRpSnoC28oWvjLeshoQz1StZ9YHM1EpcJ'
+  )
+  t.deepEqual(
+    hdnode2.privateExtendedKey(),
+    'xprv9s21ZrQH143K4KqQx9Zrf1eN8EaPQVFxM2Ast8mdHn7GKiDWzNEyNdduJhWXToy8MpkGcKjxeFWd8oBSvsz4PCYamxR7TX49pSpp3bmHVAY'
+  )
+  t.end()
```

Comment:
Could we perhaps add some comments that makes these two cases a bit clearer (i.e. `hdnode1` testing fromExtendedKey with a public key, and `hdnode2` is testing it with a private key) ?

Reply:
(Could just be added as a third arg to deepEqual)